processor/transformprocessor/README.md

```diff
@@ -70,6 +74,7 @@ processors:
         - keep_keys(resource.attributes, "service.name", "service.namespace", "cloud.region")
         - set(name, attributes["http.route"])
         - replace_match(attributes["http.target"], "/user/*/list/*", "/user/{userId}/list/{listId}")
+        - replace_pattern(resource.attributes["process.command_line"], "password\\=[^\\s]*(\\s?)", "password=***")
```

Comment:
@davidmirza408 please update the traces operations list for traces [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e2cacb0e5877c499a1dd820b7a596da93ecacd71/processor/transformprocessor/README.md?plain=1#L113) and the logs operation list [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e2cacb0e5877c499a1dd820b7a596da93ecacd71/processor/transformprocessor/README.md?plain=1#L134).  This is necessary because you added more operations to the query list.

Reply:
@TylerHelmuth done